arbitrum-docs/partials/_glossary-partial.md

```diff
@@ -112,6 +127,9 @@
 ### Delayed Inbox {#delayed-inbox}
 <p>A contract that holds <a href="/intro/glossary#parent-chain">Parent chain</a> initiated messages to be eventually included in the <a href="/intro/glossary#fast-inbox">Fast Inbox</a>. Inclusion of messages doesn't depend on the <a href="/intro/glossary#sequencer">Sequencer</a>.</p>
 
+### Dev-Tools Dashboard {#devtools-dashboard}
+<p>Web application built and maintained by <a href="/intro/glossary#offchain-labs">Offchain Labs</a> for developers and users to debug Arbitrum transactions; i.e., executing or checking the status of <a href="/intro/glossary#crosschain-message">Cross-chain message</a>s; visit it <a href="https://retryable-tx-panel-nitro.arbitrum.io/">here</a>.   </p>
```

Comment:
```suggestion
<p>Web application built and maintained by <a href="/intro/glossary#offchain-labs">Offchain Labs</a> for developers and users to debug Arbitrum transactions; i.e., executing or checking the status of <a href="/intro/glossary#crosschain-message">Cross-chain messages</a>; visit it <a href="https://retryable-dashboard.arbitrum.io/">here</a>.   </p>
```

Same as above for the 's'. 
As per the URL, I think the canonical one is a bit different and doesn't have the "-nitro" part in it.